### PR TITLE
feat(ui): dashboard pages layout component

### PR DESCRIPTION
**Pull request by g10**
`PageLayout` component for dashboard pages

**Fixes** # MON-19322

- [x] New functionality (non-breaking change)

### CODE REVIEW

File: centreon/packages/ui/src/components/Layout/PageLayout/PageLayout.styles.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    padding: theme.spacing(3, 4, 5)
+  },
+  pageLayoutHeader: {
+    '[data-variant="fixed-header"] &': {
+      backgroundColor: theme.palette.layout.header.background,
+      borderBottom: `1px solid ${theme.palette.layout.header.border}`,
+      padding: theme.spacing(5, 4, 2),
+      position: 'sticky',
+      top: 0,
+      zIndex: 100
```
Comment (Thebarda) [inline centreon/packages/ui/src/components/Layout/PageLayout/PageLayout.styles.ts:25]: Use zIndex values from MUI

File: centreon/packages/ui/src/screens/dashboard/DashboardsOverview.stories.tsx [hunk 2]
```diff
@@ -150,101 +150,105 @@ const DefaultView = (args): JSX.Element => {
...
-          <Modal.Body>
-            <DashboardForm
-              labels={DashboardFormDefaultStory!.args!.labels!}
-              resource={dialogState.item || undefined}
-              variant={dialogState.variant}
-              onCancel={() =>
+        </PageLayout.Actions>
+
+        <DataTable isEmpty={dataDashboards.length === 0}>
+          {dataDashboards.length === 0 ? (
```
Comment (Thebarda) [inline centreon/packages/ui/src/screens/dashboard/DashboardsOverview.stories.tsx:156]: extract it into a specific variable

File: centreon/packages/ui/src/Dashboard/utils.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { lt } from 'ramda';
-import { Layout } from 'react-grid-layout';
+import { Layout } from 'react-grid-Layout';
```
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/Dashboard/utils.ts:2]: 👍 noticed the renaming refactor went wrong on several places

File: centreon/packages/ui/src/components/Layout/AreaIndicator.styles.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    '&[data-depth="1"]': {
+      '& label:first-of-type': {
+        '&:before': {
+          content: '"+ "'
+        },
+        left: theme.spacing(10)
+      }
+    },
+
+    backgroundColor: 'rgba(151, 71, 255, .1)',
```
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/components/Layout/AreaIndicator.styles.ts:24]: any specific reason ? `rgba()` allows to tweak the alpha easier therefore used here

_...note, this is a helper component to illustrate areas in stories_

File: centreon/packages/ui/src/components/Layout/AreaIndicator.styles.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    // border: '1.5px dashed rgba(151, 71, 255, .5)',
+    borderRadius: '4px',
```
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/components/Layout/AreaIndicator.styles.ts:27]: border ≠ spacing

File: centreon/packages/ui/src/screens/dashboard/DashboardsOverview.stories.tsx (hunk 2 above)
Comment (g10) [reply centreon/packages/ui/src/screens/dashboard/DashboardsOverview.stories.tsx:156]: this is a storybook screen mockup, no production code

File: centreon/packages/ui/src/Dashboard/utils.ts
Comment (Thebarda) [inline centreon/packages/ui/src/Dashboard/utils.ts:1]: Is it an issue with Jetbrains. Because nothing has changed here :/
tab vs space issue?

Comment (g10) [reply centreon/packages/ui/src/Dashboard/utils.ts:1]: it's the line separator that has changed from CRLF → LF

File: centreon/packages/ui/src/components/Layout/AreaIndicator.styles.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { makeStyles } from 'tss-react/mui';
+
+export const useStyles = makeStyles()((theme) => ({
+  areaIndicator: {
+    '& .areaIndicator': {
+      left: theme.spacing(10)
+    },
+    '& label:first-of-type': {
+      fontSize: '0.75rem',
```
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/components/Layout/AreaIndicator.styles.ts:9]: fyi. this is a helper component, not linked to the theme

File: centreon/packages/ui/src/Dashboard/utils.ts
Comment (Thebarda) [reply centreon/packages/ui/src/Dashboard/utils.ts:1]: I think you should update your IDE to CRLF

**Review by Thebarda — APPROVED** [review]

**Comment by Thebarda** [comment]
Can you complete the setup for chromatic pls, if it's possible?

**Review by bdauria — APPROVED** [review]

**Pull request merged**